.github/workflows/conan-package.yml

```diff
@@ -0,0 +1,152 @@
+---
+name: conan-package
```

Comment:
Isn't a better name here "Create Conan Package". The name is intended to be human readable, right?